Sources/Build/CMakeLists.txt

```diff
@@ -25,9 +25,11 @@ target_link_libraries(Build PUBLIC
 set_target_properties(Build PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
 
-if(CMAKE_SYSTEM_NAME STREQUAL Windows)
 install(TARGETS Build
```

Comment:
This is not correct. We should only install swift-{build, package, test, run} binaries in the toolchain.

Reply:
This is under your control: the component system allows you to control that.  By default everything is installed, but, `build-script-impl` can specify the `bins` component to only install the tools.

Reply:
Sure but it's not correct to declare that these libs can be installed in the toolchain. These are purely intermediates.

Reply:
They are not intermediates when built shared.  So, having the install target with the ability to install them allows both use cases.